src/Views/HardwareView.vala

```diff
@@ -280,6 +282,23 @@ public class About.HardwareView : Gtk.Grid {
 
         return 0;
     }
+
+    private string get_storage_type () {
+        string storage = "";
+        try {
```

Comment:
There isn't an Errorable method here, so we actually get a warning from the compiler that the catch clause is unreachable. So this can be removed